src/detect.c

```diff
@@ -199,6 +199,8 @@
 
 #include "runmodes.h"
 
+#include <glob.h>
```

Comment:
Lets not bother with a 'not found'-path at first. I will run the new PR through my QA and if that passes we can assume glob.h is available everywhere.